src/network/protocol/messages/getblocks.zig

```diff
@@ -0,0 +1,184 @@
+const std = @import("std");
+const native_endian = @import("builtin").target.cpu.arch.endian();
+const protocol = @import("../lib.zig");
+
+const ServiceFlags = protocol.ServiceFlags;
+
+const Endian = std.builtin.Endian;
+const Sha256 = std.crypto.hash.sha2.Sha256;
+
+const CompactSizeUint = @import("bitcoin-primitives").types.CompatSizeUint;
+
+/// GetblocksMessage represents the "getblocks" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#getblocks
+pub const GetblocksMessage = struct {
+
+    version: i32,
+    hash_count: u64,
```

Comment:
You can entirely remove the `hash_count` field. As `header_hashes.len` is the same exact information

Reply:
right, this is useless